examples/hello-world-hierarchy/cr/hello-world-cr.yaml

```diff
@@ -1,15 +1,16 @@
 apiVersion: eunomia.kohls.io/v1alpha1
 kind: GitOpsConfig
 metadata:
-  name: hello-world-hiera
+  name: hello-world-hierarchy
 spec:
   templateSource:
     uri: https://github.com/KohlsTechnology/eunomia
     ref: master
     contextDir: examples/hello-world-helm/templates
   parameterSource:
-    ref: master
-    contextDir: examples/hello-world-hiera/hierademo
+    #ref: master
+    ref: basic_hiera
```

Comment:
Before merging, this needs to be cleaned up back to `master`, yes?

Reply:
Oops. Yes that definitely will need to get cleaned up once I'm done.